### ISSUE TITLE
Fix back button on search query page/word page from going to input page to previous page.

### ISSUE THREAD

**Issue by helloitsrufio** [helloitsrufio/Southern-Paiute-Talking-Dictionary#10]
Fix back button on search query page/word page from going to input page to previous page.

https://www.loom.com/share/68bb47dc9f53461d976af334fca9d02e

**Comment by devlark** [comment]
I cant really test this since it requires an entry to be added first, but my cloudinary thingy is not recognizing my api key.
So, instead while i figure that out, here is some a brief explanation of what I think is causing this bug:
 -  as is well pointed in the comment above this line, the link back does not have the search params pass dynamically.
  https://github.com/helloitsrufio/Southern-Paiute-Talking-Dictionary/blob/a744c0e663bac8e5b414a8d6a1f822d93931fa52/views/wordPage.ejs#L35
- so adding those missing params is one solution. Another is to instead of passing params is to somehow, add an eventlistener to that <a> and just do a history.back like: `<a onclick="history.back()">Go back</a>`
- another solution is to remove the back button altogether, and let the user use the browser native back button behavior.

Once my cloudinary thingy starts working I'll do more testing, development and magic to fix this.

**Comment by devlark** [comment]
Soooo.... 6 months later.... i finally got my cloudinary stuff working (it turned out i was missing a .env in the config folder lol)
Ive also found a solution:

First replace this with `id="go-back-to-results` (this id can be anything)
https://github.com/helloitsrufio/Southern-Paiute-Talking-Dictionary/blob/a744c0e663bac8e5b414a8d6a1f822d93931fa52/views/wordPage.ejs#L35
like this:
```html
<a id="go-back-to-results">
```
then after this line:
https://github.com/helloitsrufio/Southern-Paiute-Talking-Dictionary/blob/a744c0e663bac8e5b414a8d6a1f822d93931fa52/views/wordPage.ejs#L97
we can add the next:
```javascript
<script>
      const goBackToResults = document.getElementById("go-back-to-results");
     
      if (goBackToResults) {
        goBackToResults.addEventListener("click", () => {
          if (window.history.length > 1) {
            window.history.back();
          } else {
            window.location.href = "/";
          }
        });
      }
</script>
```
basically, we're checking if there is any history in the browser history API, if exists then use it (duh) and go back(), else just redirect to index. We're doing this in case a user visits the site from /word?search=asdf for the first time, but if they have search for a word and then want to go back, it should work too.

So yeah.... I'll make a pull request soon™

**Comment by devlark** [comment]
added pr #12 with solution
